aten/src/ATen/core/functional.h

```diff
@@ -13,14 +13,23 @@ namespace c10 {
 //
 // No explicit template parameters are required.
 
+
```

Comment:
Consider adding a test for new code

https://github.com/pytorch/pytorch/tree/master/c10/test/util

Reply:
Thanks, didn't see any existing tests and wasn't sure where to put them